packages/devextreme-cli/src/applications/application.react.js

```diff
@@ -62,7 +64,7 @@ const create = async(appName, options) => {
 
     const commandArguments = [`-p=create-vite@${latestVersions['create-vite']}`, 'create-vite', appName];
 
-    commandArguments.push(`--template react${templateOptions.isTypeScript ? '-ts' : ''}`);
+    commandArguments.push(`--template react${transpiler === 'swc' ? '-swc' : ''}${templateOptions.isTypeScript ? '-ts' : ''}`);
```

Comment:
What about changing the condition ${transpiler === 'swc' ? '-swc' : ''} to ${'-'transpiler} and setting the value as '' for the Babel option in the transpiler module?